File: 58-length-of-last-word/58-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        sList = s.split(" ")
        res = []
        for e in sList:
            if e == "":
                continue
            else:
                res.append(e)
        return len(res[len(res) -1])
        

File: 58-length-of-last-word/58-length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let sArray = s.split(" ")
    
    const res = []
    
    for (let e of sArray){
        if (e === "") continue
        else (res.push(e))
    }
    
    return res[res.length - 1].length
};